src/app/loadout/LoadoutDrawerContents.tsx

```diff
@@ -205,3 +212,30 @@ function fillLoadoutFromEquipped(
     }
   }
 }
+
+async function fillLoadoutFromUnequipped(
+  loadout: Loadout,
+  stores: DimStore[],
+  add: (item: DimItem, e?: MouseEvent, equip?: boolean) => void
+) {
+  if (!loadout) {
+    return;
+  }
+  const dimStore =
+    (loadout.classType !== DestinyClass.Unknown &&
+      stores.find((s) => s.classType === loadout.classType)) ||
+    getCurrentStore(stores)!;
+
+  const items = dimStore.items.filter(
+    (item) =>
+      item.bucket.type !== 'Class' &&
+      itemCanBeInLoadout(item) &&
+      fromEquippedTypes.includes(item.type)
+  );
+
+  for (const item of items) {
+    if (fromEquippedTypes.includes(item.type)) {
```

Comment:
You've already filtered the items on this, so you don't need to do it again.

Reply:
I see what you are saying. Thank you.